packages/twentytwenty-theme/src/components/search/search-modal.js

```diff
@@ -19,6 +21,12 @@ const SearchModal = ({ state, actions, libraries }) => {
   const inputRef = useRef();
   const containerRef = useRef();
 
+  const transitions = useTransition(isSearchModalOpen, null, {
```

Comment:
- I am wondering (out of ignorance) if we can achieve the same animation with CSS and avoid `useTransition`?
- In that case: does it make sense or it is harder to create and/or harder to understand?

Reply:
I think both @DAreRodz and @orballo used to do this type of animations using Emotion's `keyframes`:
https://emotion.sh/docs/keyframes

Is that right guys?